subprojects/android-test/toast-rule/src/main/kotlin/com/avito/android/rule/internal/ProxyToastChecks.kt

```diff
@@ -0,0 +1,110 @@
+package com.avito.android.rule.internal
+
+import android.content.Context
+import android.view.View
+import android.widget.Toast
+import androidx.annotation.StringRes
+import androidx.test.espresso.matcher.ViewMatchers
+import com.avito.android.rule.ToastChecks
+import com.avito.android.test.waitFor
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.MatcherAssert
+import org.hamcrest.Matchers
+import org.hamcrest.Matchers.empty
+import org.hamcrest.Matchers.not
+import org.hamcrest.TypeSafeMatcher
+
+internal class ProxyToastChecks(
+    private val proxyToast: MockProxyToast,
+    private val textAccessor: ToastTextAccessor,
+) : ToastChecks {
+
+    private val longToastDuration = 3500L
```

Comment:
Any chance to get value from android resources?

Reply:
Found it in a hidden class. 
We could access it only through reflection, but it will be version-specific.
I think it's too much for this knowledge.